### ISSUE TITLE
Discuss Fmod vs Mod

### ISSUE THREAD

**Issue by texone** [TheFuseLab/VL.Fuse#25]
Should we just use the glsl style Mod which is propably what people expect any advantage on using fmod?

**Comment by everyoneishappy** [comment]
I'd suggest to just provide both?  I don't know if it's advantage so much as just slightly different behavior?

I've already got it in the FuseCommonSDF header, but could move it to FuseCoreMath
`	// glsl style mod
	#ifndef mod
	#define mod(x, y) (x - y * floor((x) / y))
	#endif`

**Comment by everyoneishappy** [comment]
Ah they are both there already (mod is patched)

**Comment by texone** [comment]
Yes I am aware already used mod question is should we keep fmod or just remove it or add a help patch explaining both and add proper documention to both nodes to make sure which one to use
